docs/the-button.md

```diff
@@ -11,15 +11,16 @@ The `/etc/pisound.conf` file allows mapping these actions:
 | CLICK_1 | Button was clicked once. | [start_puredata.sh](https://github.com/BlokasLabs/pisound/blob/master/scripts/pisound-btn/start_puredata.sh) |
 | CLICK_2 | Button was double-clicked. | [stop_puredata.sh](https://github.com/BlokasLabs/pisound/blob/master/scripts/pisound-btn/stop_puredata.sh) |
 | CLICK_3 | Button was triple-clicked. | [toggle_wifi_hotspot.sh](https://github.com/BlokasLabs/pisound/blob/master/scripts/pisound-btn/toggle_wifi_hotspot.sh) |
-| CLICK_OTHER | Button was clicked between 4 and 8 times. | [do_nothing.sh](https://github.com/BlokasLabs/pisound/blob/master/scripts/pisound-btn/do_nothing.sh) |
+| CLICK_OTHER | Button was clicked between 4 and 8 times.  The upper limit is altered by the configuration file entry `CLICK_COUNT_LIMIT` or by the command line option [--click-count-limit](#command-line-options).  If the --no-defaults options is | [do_nothing.sh](https://github.com/BlokasLabs/pisound/blob/master/scripts/pisound-btn/do_nothing.sh) |
```

Comment:
Looks like the last sentence `If the --no-defaults options is` is incomplete. :)